### ISSUE TITLE
end-to-end could report back failure of the npx command

### ISSUE THREAD

**Issue by eiswind** [leptos-rs/cargo-leptos#90]
When we run end-to-end (I'm using npx cypress run) and a test fails, the cargo leptos command could report back the failure from the spawned process to make a possible ci pipeline fail.
I tried to extract the ExitStatus in try_run but so far this gets a mess. I guess I'm not form enough with rust to create  an elegant solution.

What I tried is like

```rust
let mut exit_status : Option<ExitStatus> =  None;
tokio::select! {
      _ = int.recv() => {},
     result = process.wait() => {
        if let Ok(status) = result {
            exit_status = Some(status);
        }
     }
}
```
Is there a better way to get hold of the exit status?

**Comment by eiswind** [comment]
After reading the docs with a little more care I worked out

```rust
let result : Option<ExitStatus> = tokio::select! {
      _ = int.recv() => None,
     result = process.wait() => Some(result?)
};
if let Some(exit_status) = result {
    if let Some(exit_code) = exit_status.code() {
        if exit_code != 0 {
            return Err(anyhow!("Command terminated with exit code {}", exit_code))
        }
    }
}
```
I'll check this out in my ci pipeline as soon as I can find time.

**Comment by akesson** [comment]
Closed by #93
